spec/index.bs

```diff
@@ -1799,20 +1799,90 @@ dictionary GPUBindGroupLayoutEntry {
+
+    // Used for storage texture bindings.
+    GPUTextureFormat storageTextureFormat;
 };
 </script>
 
-  * {{GPUBindGroupLayoutEntry/binding}}:
-    A unique identifier for a resource binding within a {{GPUBindGroupLayoutEntry}}, a corresponding {{GPUBindGroupEntry}}, and shader stages.
+<dl dfn-type=dict-member dfn-for=GPUBindGroupLayoutEntry>
+    : <dfn>binding</dfn>
+    ::
+        A unique identifier for a resource binding within a
+        {{GPUBindGroupLayoutEntry}}, a corresponding {{GPUBindGroupEntry}},
+        and shader stages.
+
+    : <dfn>visibility</dfn>
+    ::
+        A bitset of the members of {{GPUShaderStage}}.
+        Each set bit indicates that a {{GPUBindGroupLayoutEntry}}'s resource
+        will be accessible from the associated shader stage.
+
+    : <dfn>hasDynamicOffset</dfn>
+    ::
+        If the {{GPUBindGroupLayoutEntry/type}} is
+        {{GPUBindingType/"uniform-buffer"}},
+        {{GPUBindingType/"storage-buffer"}}, or
+        {{GPUBindingType/"readonly-storage-buffer"}},
+        this indicates whether a binding requires a dynamic offset.
+
+        Otherwise, it must be `false`.
+
+    : <dfn>viewDimension</dfn>
+    ::
+        If the {{GPUBindGroupLayoutEntry/type}} is
+        {{GPUBindingType/"sampled-texture"}},
+        {{GPUBindingType/"readonly-storage-texture"}}, or
+        {{GPUBindingType/"writeonly-storage-texture"}},
+        this defaults to {{GPUTextureViewDimension/"2d"}}
+        and is the required {{GPUTextureViewDescriptor/dimension}}
+        of a texture view bound to this binding.
+
+        Otherwise, it must be `undefined`.
+
+    : <dfn>textureComponentType</dfn>
+    ::
+        If the {{GPUBindGroupLayoutEntry/type}} is
+        {{GPUBindingType/"sampled-texture"}},
+        this defaults to {{GPUTextureComponentType/"float"}}
+        and is the required component type of the {{GPUTextureViewDescriptor/format}}
+        of a texture view bound to this binding.
+
+        Otherwise, it must be `undefined`.
+
+    : <dfn>multisampled</dfn>
+    ::
+        If the {{GPUBindGroupLayoutEntry/type}} is
+        {{GPUBindingType/"sampled-texture"}},
+        this indicates whether a binding must have a sample count greater than 1.
+
+        Otherwise, it must be `false`.
+
+    : <dfn>storageTextureFormat</dfn>
+    ::
+        If the {{GPUBindGroupLayoutEntry/type}} is
+        {{GPUBindingType/"readonly-storage-texture"}}
+        or {{GPUBindingType/"writeonly-storage-texture"}},
+        this is the required {{GPUTextureViewDescriptor/format}}
+        of a texture view bound to this binding.
+
+        Otherwise, it must be `undefined`.
+</dl>
 
-  * {{GPUBindGroupLayoutEntry/visibility}}:
-    A bitset of the members of {{GPUShaderStage}}. Each set bit indicates that a {{GPUBindGroupLayoutEntry}}'s resource will be accessible from the
-    associated shader stage.
+Note:
+{{GPUBindGroupLayoutEntry/viewDimension}} and {{GPUBindGroupLayoutEntry/multisampled}}
```

Comment:
note `multisampled` is also important for pipeline creation, since we are able to match the `Texture2DMS` from the shader, for example, with the binding layout, at pipeline creation time.